src/attributes/limits.md

```diff
@@ -4,10 +4,12 @@ The following [attributes] affect compile-time limits.
 
 ## The `recursion_limit` attribute
 
-The *`recursion_limit` attribute* may be applied at the crate level to set the
+The *`recursion_limit` attribute* may be applied at the [crate] level to set the
 maximum depth for potentially infinitely-recursive compile-time operations
 like auto-dereference or macro expansion. It uses the [_MetaNameValueStr_]
```

Comment:
Add an example (after macros) for auto-dereference since you reference it, e.g.
```rust
#![recursion_limit = "1"]

fn main() {
    (|_: &u8| {})(&&1);
}
```